net/gensync/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+
+github.setup            nislab gensync 1.0.4 v
+git.branch              vanilla
+
+name                    gensync
+version                 1.0
```

Comment:
Why are you manually setting the version to `1.0` when it appears that it should correctly be `1.0.4` as according to the `github.setup` line?